src/main/java/io/fabric8/maven/docker/config/BuildImageConfiguration.java

```diff
@@ -190,6 +190,9 @@ public class BuildImageConfiguration implements Serializable {
     @Parameter
     private Boolean skipPush;
 
+    @Parameter
+    private Boolean skipTag;
```

Comment:
Umm, is this new field really needed here? Do you want to skip tag at image level?

Reply:
> Do you want to skip tag at image level?

When using the buildx to build the multi-arch with push action, I do not want to push tag.

Right now, this PR supports skip tag at image level.

Reply:
okay, I see your point now. `skipTag` configuration option is not available in BuildXService